File: C/EX18.c
/*Calcula IMC baseado no sexo feminino ou masculino*/
#include<stdio.h>
#include<stdlib.h>

int main(int argc, char const *argv[])
{
    float peso, altura, imc;
    char sexo;

    printf("Informe \nPeso:");
    scanf("%f", &peso);

    printf("\nInforme \nAltura:");
    scanf("%f", &altura);

    imc = peso / (altura*altura);

    printf("Informe \nSexo:");
    scanf("%s", &sexo);

   // imc = peso / (altura*altura);

    if (sexo == 'f')  
    {
        if (imc >= 24)
        {
            printf("acima do peso");
        }
        
        else if (imc < 19)
        {
            printf("Abaixo do peso");
        }
        
        else
        {
            printf("Peso ideal");
        }
        
        
    }
    
    else
    {
        if (imc >= 25)
        {
            printf("acima do peso");
        }
        
        else if (imc < 20)
        {
            printf("Abaixo do peso");
        }
        
        else
        {
            printf("Peso ideal");
        }
    }
    
    
    return 0;
}

File: Python/EX18.py
#imc para homens e mulheres

peso = float(input("Peso: "))
altura = float(input("Altura: "))

imc = peso / (altura**2)

sexo = input("Sexo: m-masculino e f-feminino: ")

if sexo == 'f':
    
    if imc >=25:
        print("Acima do peso.")
    
    elif imc < 20:
        print("Abaixo do peso.")
    
    else:
        print("Peso ideal")

else:
    if imc >= 24:
        print("Acima do peso.")
    
    elif imc < 19:
        print("Abaixo do peso.")
    
    else:
        print("Peso ideal")
